package.json

```diff
@@ -305,6 +306,8 @@
     "markdown-it-link-attributes": "^4.0.0",
     "moment": "~2.24.0",
     "moment-timezone": "^0.5.34",
+    "octokit": "^1.7.1",
```

Comment:
Since `octokit` includes the API client, we should be able to get rid of the `@octokit/rest` package that we have installed. A couple files might need to be refactored though